cmd/gossamer/configcmd_test.go

```diff
@@ -34,7 +34,6 @@ import (
 	"github.com/ChainSafe/gossamer/node/gssmr"
 	"github.com/ChainSafe/gossamer/tests"
 
-	log "github.com/ChainSafe/log15"
```

Comment:
Why use fmt instead of log in this file?

Reply:
I am pretty sure we want to avoid using logs in tests. @noot pointed this out to me when I was attempting to update the format of these logs.

Reply:
yeah we should be using t.Log or fmt, doesn't really make sense to use the logger in tests